atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -62,7 +63,8 @@ private AtlasDbConstants() {
 
     public static final TableReference PARTITION_MAP_TABLE = TableReference.createWithEmptyNamespace("_partition_map");
     public static final byte[] EMPTY_TABLE_METADATA = {}; // use carefully
-    public static final byte[] GENERIC_TABLE_METADATA = new TableMetadata().persistToBytes();
+    public static final byte[] GENERIC_TABLE_METADATA = new TableMetadata(TableMetadataPersistence.LogSafety.SAFE)
+            .persistToBytes();
```

Comment:
I've checked for uses of this and it is safe, but I think we should do one of

1. leave the old `GENERIC_TABLE_METADATA` as is (maybe with a comment that it exists to avoid breaking the API), and add a new constant `GENERIC_SAFE_TABLE_METADATA` which is this constant.
2. make a note that this is a dev break in the release notes (it is, because `GENERIC_TABLE_METADATA` used to be unsafe)

Reply:
I dont think anyone uses `GENERIC_TABLE_METADATA` directly, checked the large internal product too, so going with option 2.

Reply:
Yep, I should have mentioned that I checked internally too!